### PR TITLE
Ensdf processing alphad

### PR DESCRIPTION
**Pull request by howland**
This PR adds the ALPHAD and GABS ensdf analysis programs from the nndc to PyNE.
http://www.nndc.bnl.gov/nndcscr/ensdf_pgm/analysis/

@scopatz: can you look over the fortran compiler flag changes we discussed?
@rachelslaybaugh

### CODE REVIEW

File: pyne/ensdf_processing.py [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+    else:
+        inp = inp + 'N' + '\n'
+    proc.stdin.write(inp)
+    proc.communicate()[0]
+    proc.stdin.close()
+    return inputdict_unchecked
+
+def gabs(inputdict_unchecked):
+    """
+    This function ...
```
Comment (scopatz) [inline pyne/ensdf_processing.py]: `...`?

File: setup.py [hunk 2]
```diff
@@ -433,6 +440,9 @@ def main():
         pynepath = "${HOME}/.local/python2.7/site-packages"
     libpath = abspath(joinpath(pynepath, '..', '..', '..'))
     binpath = abspath(joinpath(libpath, '..', 'bin'))
+    copy_ensdf_executables(pynepath)
+    print(pynepath)
+    print(libpath)
```
Comment (scopatz) [inline setup.py]: Probably safe to remove these print statements.

**Comment by scopatz** [comment]
Looks good to me overall @howland! Still seeing some CI failures.  Can you dig in and see what is causing them, please?

**Comment by howland** [comment]
@scopatz In setup.py I copy the cmake generated alphad executable from pyne/build to the install directory using pynepath (to locate the install directory):

Edit:

Fixed, all tests passing

File: setup.py (hunk 2 above)
Comment (howland) [inline setup.py]: Sorry, those were in to help me debug what was going on with the failing tests.  I'll make sure to remove them as soon as I get the tests passing.

**Comment by rachelslaybaugh** [comment]
Looks like once all the parts and pieces are in here we'll need to hook documentation up to the user guide (and in with the other ensdf stuff), but that seems premature at this point. All of this looks ok and the CI is clean.

**Pull request merged**